build/crd-samples/devices/modbusrtu-device-instance.yaml

```diff
@@ -51,10 +51,10 @@ status:
       reported:
         metadata:
           timestamp: '1550049403598'
-          type: integer
+          type: int
```

Comment:
I am not quite sure if we have any testcase to verify this.

Reply:
Yes, there's no automated test to verify mappers. It's on the TODO list.

Reply:
file `edge/pkg/devicetwin/dtcommon/util.go` function `ValidateValue` check the metadata type, it should be `int`, but not `integer`